Dockerfile

```diff
@@ -14,6 +14,7 @@ WORKDIR /usr/src/app
 
 COPY package.json /usr/src/app/
 COPY bat-utils/package.json /usr/src/app/bat-utils/
+RUN npm install -g npm@5.10
```

Comment:
we should put this above the two `COPY` statements

the reasoning is that we're more likely to change the `package.json` file than update the version of npm. with the current order, any time we change `package.json` we will have to reinstall npm